### PR TITLE
Ambient Security Architecture docs

### PR DESCRIPTION
**Pull request by darshannere**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [x] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [x] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by dhawton** [comment]
@darshannere Looks like the lint test is failing, couple minor easy to fix things. With the repo checked out locally, you can always run `make lint` to verify all passes before pushing. Also when you have a second, please fill out the CLA. It's a requirement before your PR can merge.

**Comment by darshannere** [comment]
@dhawton @linsun I have solved all the errors. Can you review the pr and let me know if it is good.

**Comment by linsun** [comment]
cc @christian-posta wanted to make sure he is good at incorporating these diagrams to istio.io doc.

File: content/en/docs/ops/ambient/architecture/index.md [hunk 1]
```diff
@@ -86,3 +86,51 @@ spec:
 {{< /text >}}
 
 ## Security
+
+{{< image link="./ambient-layers.png" caption="Layering of ambient mesh data plane" >}}
+
+To recap, Istio ambient mesh introduces a layered mesh data plane with a secure overlay responsible for transport security and routing, that has the option to add L7 capabilities for namespaces that need them.
```
Comment (linsun) [inline content/en/docs/ops/ambient/architecture/index.md:92]: what do you mean by recap here?

File: content/en/docs/ops/ambient/architecture/index.md [hunk 2]
```diff
@@ -86,3 +86,51 @@ spec:
...
+To understand more, please see the [announcement blog](/blog/2022/introducing-ambient-mesh/) and the [getting started blog](/blog/2022/get-started-ambient).
+The secure overlay consists of a node-shared component, the ztunnel, that is responsible for L4 telemetry and mTLS which is deployed as a DaemonSet.
+The L7 layer of the mesh is provided by waypoint proxies, full L7 Envoy proxies that are deployed per identity/workload type.
```
Comment (linsun) [inline content/en/docs/ops/ambient/architecture/index.md:95]: also per namespace

**Review by craigbox — COMMENTED** [review]
Hey @darshannere, can you take a look at the outstanding comments and we can merge this?  Thanks!

**Comment by craigbox** [comment]
@darshannere  @dhawton @linsun is there something of value here to bring forward?

**Comment by craigbox** [comment]
This is just a copy of https://istio.io/latest/blog/2022/ambient-security/.  I do think it makes sense to bring some of this content forward but not in this PR.

**Pull request closed without merging**